.azuredevops/deploy-bootstrap-serviceconnection.yaml

```diff
@@ -63,5 +63,14 @@ extends:
                 ScriptArguments: >
                   -SubscriptionName $(subscriptionName) 
                   -KeyVaultName $(ssvPlatformKeyVaultName)
-                  -Tier2ApplicationClientIdSecretName $(tier2ApplicationClientIdSecretName)                   
+                  -Tier2ApplicationClientIdSecretName $(tier2ApplicationClientIdSecretName)    
+              - displayName: Setup AzureRm Service Endpoint
+                Type: AzureCLI
+                scriptPath: 'scripts/Setup-Service-Connection.ps1'
```

Comment:
I agree we'd need a script for service connection, but why isn't there a file of service connections to setup, or similar, so it's config and declarative driven ? means we can have 1x configuration file of connections and a generic script... @ganeshg2491  @kbabigumira  @ajaysoni-cap